core/model/__init__.py

```diff
@@ -45,15 +56,22 @@ def flush(db):
         db.flush()
 
 
-def create(db, model, create_method="", create_method_kwargs=None, **kwargs):
+T = TypeVar("T")
+
+
+def create(
+    db: Session, model: Type[T], create_method="", create_method_kwargs=None, **kwargs
+) -> Tuple[T, Literal[True]]:
     kwargs.update(create_method_kwargs or {})
     created = getattr(model, create_method, model)(**kwargs)
     db.add(created)
     flush(db)
     return created, True
 
 
-def get_one(db, model, on_multiple="error", constraint=None, **kwargs):
+def get_one(
+    db: Session, model: Type[T], on_multiple="error", constraint=None, **kwargs
+) -> T | None:
```

Comment:
I added generic typing to these foundational methods because I got tired of added type hints in all the methods that use them, and guess what, I had to add defensive code ALLLL over the place so mypy stays happy now.
This added defensive code is why test coverage has also dropped in many files.
I'm too stupid for my own good sometimes.